test/unit/rules/lint-no-aria-attributes-on-unsupported-elements-test.js

```diff
@@ -0,0 +1,84 @@
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-aria-attributes-on-unsupported-elements',
+
+  config: true,
+
+  good: [
+    '<div role="button"></div>',
+    '<img role="option">',
+    '<meta data-name="foo">',
+    '<script src="noop"></script>',
+    '<html title="noop"></html>',
+  ],
+
+  bad: [
```

Comment:
After looking at these "bad" tests, I think we could abstract them a bit more, to consolidate them.